### PR TITLE
Add SIG-GCP

### LINKED ISSUE #1322: Update sigs.yaml to include SIG GCP contact info
I had to go searching kubernetes-dev@ to find the kubernetes-sig-gcp mailing list, can we get that contact info into sigs.yaml so that docs are generated and https://github.com/kubernetes/community/tree/master/sig-gcp becomes useful?

@kubernetes/sig-gcp-misc 
/kind documentation

### PR DESCRIPTION
**Pull request by abgworrall**
Updates sigs.yaml, run `make WHAT=sig-gcp generate`, to add a new SIG.
Fixes #1322

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -361,6 +361,24 @@ sigs:
...
+      A Special Interest Group for building, deploying, maintaining,
+      supporting, and using Kubernetes on the Google Cloud Platform.
+    leads:
+    - name: Adam Worrall
+      github: abgworrall
+      company: Google
+    meetings:
+    - day: Thursday
+      utc: "16:00"
+      frequency: monthly
```
Comment (cblecker) [inline sigs.yaml]: Aren't we meeting biweekly?

Comment (abgworrall) [reply sigs.yaml]: Good catch, things changed since I drafted this :)

**Pull request merged**